interview-questions.md

```diff
@@ -16,53 +16,60 @@ INSTRUCTOR EXAMPLE: What is a conditional statement?
 1. What is git? What is the difference between git and Github?
 
   Your answer:
-
-  Researched answer:
-
+Git is a software called Global Interface Technology used in the Github site. Github is the website using the git software.
+  Researched answer: The git software is used mainly for tracking changes in files. The git software helps progammers coordinate work when they are collaborating to develop source codes.
+      The Github is a repository that is cloud based, uses git software that hosts files or project for developers.
 
 
 2. Which JavaScript operators will return a Boolean value?
 
   Your answer:
 
-  Researched answer:
+The following operators will return a Boolean value:
 
+    =  - the equals sign
+    <  - the less than sign
+    >  - the more than sign
+  Researched answer: In addition to the above operators, the following operators will return Boolean value:
+          * &&  - And relational operator
+          * ||  - Not, another relational operator
+          
+          Sometimes these operators are also used in combination with one another to further narrow down some conditions.
```

Comment:
Looks like you are missing your researched answer here. It might have gotten overwritten.